### PR TITLE
ISSD-1386 | Fix Figma divergencies in the Marketplace Header

### PR DESCRIPTION
**Pull request by ocesar9**

### CODE REVIEW

**Comment by ocesar9** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html:1]: Please, format this file.

**Review by silasbsi — CHANGES_REQUESTED** [review]
I did a suggestion

**Comment by ocesar9** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html [hunk 1]
```diff
@@ -24,45 +24,45 @@
...
-						</a>
+							<a class="account-menu-item d-flex font-weight-bold p-3 text-decoration-none" href="${htmlUtil.escape(themeDisplay.getURLSignOut())}">
+								<span class="account-menu-icon align-items-center d-flex opacity-70 pr-2 pt-1">
+									[@clay["icon"] symbol="logout"/]
+								</span>
+								<span class="dropdown-option-text">
+									${languageUtil.get(locale, "sign-out","Sign Out")}
+								</span>
+							</a>
 					</div>
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html:45]: Please, format this point.

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added a suggestion

**Comment by silasbsi** [comment]
ci:forward:force

**Pull request closed without merging**